addons/base_address_city/models/res_partner.py

```diff
@@ -20,7 +20,10 @@ def _onchange_city_id(self):
     @api.model
     def _fields_view_get_address(self, arch):
         arch = super(Partner, self)._fields_view_get_address(arch)
-        if not self._context.get('no_address_format'):
+        # if contry_id has address_view_id then not overwrite view
+        company_id = self.browse(self._context.get('default_customer'))
```

Comment:
Please change by:

`company_id = self.browse(self._context.get('default_customer', []))`

To avoid traceback when `self._context.get('default_customer')` is not found